payments-core/src/main/java/com/stripe/android/networking/StripeApiRepository.kt

```diff
@@ -1164,6 +1166,99 @@ internal class StripeApiRepository @JvmOverloads internal constructor(
         }
     }
 
+    /**
+     * Creates a new Link account for the credentials provided.
+     */
+    override suspend fun consumerSignUp(
+        email: String,
+        phoneNumber: String,
+        country: String,
+        cookies: String?,
+        requestOptions: ApiRequest.Options
+    ): ConsumerSession? {
+        return fetchStripeModel(
+            apiRequestFactory.createPost(
+                consumerSignUpUrl,
+                requestOptions,
+                mapOf(
+                    "email_address" to email.lowercase(),
+                    "phone_number" to phoneNumber,
+                    "country" to country
+                ).plus(
+                    cookies?.let {
+                        listOf("cookies" to it)
```

Comment:
Where does the cookie get stored?

Reply:
It's currently not stored, but it will be managed by the link module. Will probably use [EncryptedSharedPreferences](https://developer.android.com/reference/androidx/security/crypto/EncryptedSharedPreferences).